### ISSUE TITLE
Jura S8 Recipes

### ISSUE THREAD

**Issue by bruno-sardine** [bruno-sardine/github-slideshow#3]

**Comment by bruno-sardine** [comment]
# Jura S8 Recipes

Notes:
- New users may be confused on the milk dispensing system (using time vs volume).  Take note of the Cortado and Cortadito recipes.  They require “equal amounts of milk” to the coffee.  I brewed these drinks with a scale under the cup and stopped the milk function when the amount of needed grams of milk was reached.  For Whole Milk, 15 seconds is about 3 ounces, 12 seconds is about 2 ounces and 6 seconds is about 1 ounce.
- My grind setting: 3rd position from finest grind
- Bean settings (aroma profile).  There are 5 bean icons, each with half steps, allowing for 10 levels of aroma profile (which just alters how much coffee is ground).  If I say "Bean: 3", I mean there are 3 whole icons of beans filled in, which is actually "Level 6".  Similarly, "Bean 4.5" would mean 9 half-bean icons are filled in (just shy of the max setting).
- I'm opting to use 1oz volumes for espresso and 2oz for doppio.  I believe those are correct/traditional volumes, but coming from a Nespresso, I was used to shots that are 1.5 oz; and the Jura default is also 1.5 oz.
- Milk times are with whole milk with the exception of the Cafe Latte which I use oat milk

Coffees
--
**Espresso (1oz):**
... Bean:  4
... Volume: 1 oz

**Doppio (2oz):**
I originally used the “Double Product” feature on an espresso.  While it yielded a 2oz drink, I was not sure what was happening in the background.  I expected grind, pour, grind, pour.  I think it just ground a little more coffee.  Also, my Doppio just wasted a space on the main screen.  If you really want a doppio, just do this for your recipes:
... Bean:  5 (max)
... Volume: 2 oz

**Americano (6.5oz drink)**
... Espresso
... 5.5 oz hot water

**Latte Macchiato (7.5 oz)** 
This is the default recipe from the machine
... Bean: 4
... Volume: 3oz
... Milk: 22 Seconds
... Interval: 30 Seconds

**Cafe Cubano (1oz)**
A sweet drink consisting of whipped sugar and strong espresso.  Whipped sugar rises to the top. 
Prep: when making the espresso, slide the bean to level 5 (maximum) and make the espresso in a cup not being used to serve.
… Bean: 5
... 1 espresso (in a temporary cup - e.g. paper cup)
... 2 tsp sugar
Directions: In your serving glass/cup: Mix 1 tsp of espresso with the sugar and whip for 2-3 mins.  Pour remaining coffee in cup and mix again.

**Cortado (4oz)**
2 shots of espresso (a doppio) with equal amount of steamed milk
... Bean 5
... Espresso (2oz)
... Steamed Milk (12 seconds)

**Cafe Marron  (4oz)**
A Venezuelan drink identical to the Cortado, but with a very slightly less amount of milk
... Bean 5
... Espresso (2oz)
... Steamed Milk (10 seconds)

**Cortadito (2oz)**
1 tbsp sugar + 1 shot of espresso brewed over the sugar with equal amount of steamed milk.
... 1 tbsp sugar
... 1 espresso
Quickly mix/dissolve the sugar before the milk comes out!
... Steamed Milk: 6 seconds

**Flat White (default from machine)**
... Bean: 4
... Volume: 2oz
... Steamed Milk: 14 Seconds

**Cappuccino (5.x oz)**
This is the default recipe from the machine
... Bean: 2.5
... Steamed Milk: 14 Seconds
... Volume: 3oz

**Cafe Latte (7oz)**
This is to my taste
...Bean:  10 (max)
...Steamed Oat Milk: 45 Seconds
...Volume: 1.5 oz
Note: Lattes may typically have more milk with a 7oz minimum.  My Latte uses about 6 - 6.5 oz milk

**Cafe con Leche (6 oz)**
2 tsp of sugar + 3oz coffee + equal parts milk.
… Fill cup with 2 tsp sugar
... Espresso (slide volume to 3 oz)
... Steamed Milk: 15 secs

**Cafe con Leche (Venezuelan version)**
About 25% coffee, 75% milk
... Espresso
... Bean:  4
... Volume: 1 oz
... Steamed Milk: 15 secs (about 3 oz)

**Espresso Macchiato (1 oz)**
... Espresso
... Milk Foam: 2 seconds

**Con Panna / Viennois (vee-IN-wah)**
An espresso with whipped cream
... Espresso
… Bean: 5
… Volume 2oz
... Topped with Whipped Cream (3 oz)

**Affogato (5 oz)**
Espresso brewed on top of vanilla ice cream
... Vanilla ice Cream (3 oz)
... Espresso
… Bean: 5
… Volume: 2 oz

**Lungo (3 oz)**
Less concentrated expresso.
... Espresso 
… Volume (3 oz)

**Mocha (6 oz)**
Basically a cortado with chocolate syrup
... Espresso
… Bean: 5
… Volume: 2 oz
... Steamed Milk (12 seconds)
... Chocolate Syrup (2 oz)

**Janet's Cappuccino**
... 14g / .5 oz flavored syrup
... Bean: 1
... Steamed Milk: 14 Seconds
... Volume: 3oz

Other Jura Machine Recipe Settings (not noted above)
- I wanted to keep track of the “original” recipes since I was updating some of them to create new drinks.

Ristretto: Bean 4, volume .5oz
Espresso: Bean 4, volume 1.5oz
Milk Foam: 10 seconds milk foam
Macchiato: Bean 4, milk 5 sec., volume 1.5 oz
Coffee: Bean 2.5, volume 5 oz, Grind size maximum
Cafe au lait: Bean 5, Volume 3oz, milk 10 sec.
